Makefile

```diff
@@ -86,6 +89,22 @@ clean:
 	@rm -rf $(CURDIR)/release/
 	@echo "All clean!"
 
+clean-integration:
```

Comment:
Should `clean-integration` be a dependency of `clean`?

Also `clean-integration` should be a phony target.

Reply:
> Should clean-integration be a dependency of clean?

They technically aren't related but I also don't see why not

> Also clean-integration should be a phony target.

Ah yeah forgot

Reply:
For completeness, yeah my thinking was from a devex perspective, I expect `make clean` to fully clean everything from a project on my local system. Maybe that is opinionated but I don't think it is unreasonable either. Maybe you could argue for a deep clean but we can keep it simple unless it becomes a pain. :P